temporalcli/commandsmd/commands.md

```diff
@@ -1,487 +1,1185 @@
-is not passed, the 'default' environment is used.
+```
+temporal env set \
+    --env prod \
+    --key address \
+    --value production.f45a2.tmprl.cloud:7233
+```
 
-### temporal env delete: Delete an environment or environment property.
+Each environment is isolated. Changes to "prod" presets won't affect "dev".
 
-`temporal env delete --env environment [-k property]`
+For easiest use, set a `TEMPORAL_ENV` environmental variable in your shell.
+The Temporal CLI checks for an `--env` option first, then checks the shell.
+If neither is set, most commands use `default` environmental presets if
+they are available.
 
-Delete an environment or just a single property:
+### temporal env delete: Delete an environment or environment property
 
-`temporal env delete --env prod`
-`temporal env delete --env prod -k tls-cert-path`
 
-If the environment is not specified, the `default` environment is deleted:
+Remove an environment entirely or remove a key-value pair within an
+environment. If you don't specify an environment (with --env or by setting
+the TEMPORAL_ENV variable), this updates the default environment. 
+For example:
+
+```
+temporal env delete --env YourEnvironment
+```
 
-`temporal env delete -k tls-cert-path`
+or
+
+```
+temporal env delete --env prod  --key tls-key-path
+```
 
 <!--
 * maximum-args=1
 -->
 
 #### Options
 
-* `--key`, `-k` (string) - The name of the property.
-
-### temporal env get: Print environment properties.
+* `--key`, `-k` (string) -
+  Property name.
 
-`temporal env get --env environment`
+### temporal env get: Show environment properties
 
-Print all properties of the 'prod' environment:
-
-`temporal env get prod`
+List the properties for a given environment:
 
 ```
-tls-cert-path  /home/my-user/certs/client.cert
-tls-key-path   /home/my-user/certs/client.key
-address        temporal.example.com:7233
-namespace      someNamespace
+temporal env get --env YourEnvironment
 ```
 
 Print a single property:
 
-`temporal env get --env prod -k tls-key-path`
-
 ```
-tls-key-path  /home/my-user/certs/cluster.key
+temporal env get --env YourEnvironment --key YourPropertyKey
 ```
 
-If the environment is not specified, the `default` environment is used.
+If you don't specify an environment name, with `--env` or by setting
+`TEMPORAL_ENV` as an environmental variable in your shell, this command
+lists properties in the `default` environment.
 
 <!--
 * maximum-args=1
 -->
 
 #### Options
 
-* `--key`, `-k` (string) - The name of the property.
+* `--key`, `-k` (string) -
+  Property name.
 
-### temporal env list: Print all environments.
+### temporal env list: List environments
 
-List all environments.
+List the environments you have set up on your local computer. The output 
+enumerates the environments stored in the Temporal environment file on
+your computer ("$HOME/.config/temporalio/temporal.yaml").
```

Comment:
```suggestion
List the environments you have set up on your local computer.
Environments are stored in "$HOME/.config/temporalio/temporal.yaml".
```

(Sorry I'm not getting the wrapping exactly right on these suggestions; hard to do in GitHub's UI)

Reply:
I'll wrap. No worries.